kore/src/Kore/Repl.hs

```diff
@@ -200,7 +203,9 @@ runRepl
     makeRuleIndex n _ = RuleIndex (Just n)
 
     firstClaim :: Claim claim => claim
-    firstClaim = maybe (error "No claims found") id $ listToMaybe claims'
+    firstClaim =
+        maybe (error "No claims found") id
```

Comment:
I would rather use `(!!)` with the index inside `firstClaimIndex` to avoid duplication.